src/coreComponents/physicsSolvers/inducedSeismicity/inducedSeismicityFields.hpp

```diff
@@ -0,0 +1,170 @@
+ */
+namespace fields
+{
+
+namespace inducedSeismicity
+{
+
+DECLARE_FIELD( t_a,
+               "t_a",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Dieterich constitutive relaxation time of seismicity rate" );
+
+DECLARE_FIELD( aSigma,
+               "aSigma",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Product of direct effect parameter a and initial normal stress" );
+
+DECLARE_FIELD( pressureRate,
+               "pressureRate",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Pore pressure rate");
+
+DECLARE_FIELD( initialNormalStress,
+               "initialNormalStress",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Initial normal stress acting on the fault");
+
+DECLARE_FIELD( normalStress,
+               "normalStress",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Normal stress acting on the fault");
+
+DECLARE_FIELD( normalStress_n,
+               "normalStress_n",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Normal stress acting on the fault at the previous converged time step");
+
+DECLARE_FIELD( normalStressRate,
+               "normalStressRate",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Normal stress rate acting on the fault");
+               
+DECLARE_FIELD( initialShearStress,
+               "initialShearStress",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Initial shear stress acting on the fault");
+
+DECLARE_FIELD( shearStress,
+               "shearStress",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Shear stress acting on the fault");
+
+DECLARE_FIELD( shearStress_n,
+               "shearStress_n",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Shear stress acting on the fault at the previous converged time step");
+
+DECLARE_FIELD( shearStressRate,
+               "shearStressRate",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Shear stress rate acting on the fault");
+
+DECLARE_FIELD( seismicityRate,
+               "seismicityRate",
+               array1d< real64 >,
+               0,
+               LEVEL_0,
+               WRITE_AND_READ,
+               "Seismicity rate");
+
+DECLARE_FIELD( h,
```

Comment:
I would use a longer more descriptive variable name, for readability (unless of course, everyone understands what "h" means in this context). One-letter variable names should be avoided when representing physical quantities.

Reply:
Will change to `logSeismicityRate`